src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java

```diff
@@ -46,6 +57,22 @@ public double getPitch() {
         return limelight.getEntry("ty").getDouble(0);
     }
 
+    /**
+     * Returns the estimated robot pose as a {@link RigidTransform2}.
+     *
```

Comment:
What are the units?

Reply:
Only the translation units need to be specified, since the rotation has `toDegrees()` and `toRadians()` methods, as well as `sin` and `cos` fields (all of which are unambiguous).